app/Events/User/UserUpdated.php

```diff
@@ -2,9 +2,38 @@
 
 namespace App\Events\User;
 
+use App\Models\User;
+
 /**
  * User updated event.
  */
 class UserUpdated extends AbstractUserEvent
 {
+    /**
+     * Control of the update log.
+     *
+     * @var bool "send to log" value
```

Comment:
I nomi delle variabili e i commenti dovrebbero essere un po' più esplicativi, senza essere necessariamente più lunghi.
In questo caso potrebbe essere: `whether the only updated field is just 'last_access_at'`